core/services/vrf/v2/listener_v2.go

```diff
@@ -73,7 +73,7 @@ const (
 
 	// RandomWordsRequestedV2PlusABI is the ABI of the RandomWordsRequested event
 	// for V2Plus.
-	RandomWordsRequestedV2PlusABI = "RandomWordsRequested(bytes32 indexed keyHash,uint256 requestId,uint256 preSeed,uint64 indexed subId,uint16 minimumRequestConfirmations,uint32 callbackGasLimit,uint32 numWords,bool nativePayment,address indexed sender)"
+	RandomWordsRequestedV2PlusABI = "RandomWordsRequested(bytes32 indexed keyHash,uint256 requestId,uint256 preSeed,uint256 indexed subId,uint16 minimumRequestConfirmations,uint32 callbackGasLimit,uint32 numWords,bytes extraArgs,address indexed sender)"
```

Comment:
Just wondering, is there a better way to extract this from the v2 plus coordinator abi instead?

Maybe:
```suggestion
	RandomWordsRequestedV2PlusABI = coordinatorV2PlusABI.Events["RandomWordsRequested"].Sig
```

This returns `RandomWordsRequested(bytes32,uint256,uint256,uint256,uint16,uint32,uint32,bytes,address)` right now but I think this should still work fine for abi decoding...